src/App.vue

```diff
@@ -38,54 +40,54 @@ export default Vue.extend({
         description: 'search by name',
         initiallyCollapsed: true,
         placeholder: 'test',
-        type: 'string'
+        type: StringFilter
```

Comment:
This setup makes hard to store the filter setup, this seems like a big problem to me geven the features needed, suggest to discuss this.

Reply:
Perhaps if they're registered, you can also use the name?

Reply:
I've updated the App.vue to show that yes, this actually works